components/document-processing/overview.md

```diff
@@ -44,7 +44,7 @@ The Telerik Document Processing libraries for .NET Core support the following fi
 
 ## Available NuGet Packages
 
-You can take the required packages from the [Telerik Private NuGet Feed]({%slug installation/nuget%}), your Telerik UI for Blazor [automated installation]({%slug installation/msi%}) or [zip archive]({%slug installation/zip%}).
+You can take the required packages from the [Telerik NuGet feed]({%slug installation/nuget%}), your Telerik UI for Blazor [automated installation]({%slug installation/msi%}) or [zip archive]({%slug installation/zip%}).
```

Comment:
Perhaps "the ZIP archive" or "a ZIP archive"? (Note that throughout the PR, you might want to use ZIP archive or zip archive consistently.)